forage-android/src/main/java/com/joinforage/forage/android/Utils.kt

```diff
@@ -0,0 +1,8 @@
+package com.joinforage.forage.android
+
+import kotlin.random.Random
+
+fun generateJitter(): Int {
```

Comment:
can we mark this as an `internal fun` to avoid exposing this to the client?

Reply:
Ik i mentioned generateJitter in slack (sry), can we rename this to

```kotlin
fun getJitterAmount
```

and maybe add a comment explaining why we add jitter for maintainability?

Reply:
[c5943c9](https://github.com/teamforage/forage-android-sdk/pull/94/commits/c5943c94e45dd5b967eff50e4cf70593a4c3eac7)